packages/react/error-boundary/src/ErrorBoundary.tsx

```diff
@@ -25,31 +25,29 @@ type RenderFallbackProps<ErrorType extends Error = Error> = {
 type RenderFallbackType = <ErrorType extends Error>(props: RenderFallbackProps<ErrorType>) => ReactNode;
 type IgnoreErrorType = <ErrorType extends Error = Error>(error: ErrorType) => boolean;
 
-type ErrorBoundaryProps<ErrorType extends Error = Error> = PropsWithRef<
-  PropsWithChildren<{
-    /**
-     * @description 발생할 수 있는 error에 대한 기준값으로 이 값이 변경되면 error를 초기화합니다.
-     */
-    resetKeys?: unknown[];
-    onReset?(): void;
-    renderFallback: RenderFallbackType;
-    onError?(error: ErrorType, info: ErrorInfo): void;
-    /**
-     * @description 이 ErrorBoundary Context에서 처리하지 않고 throw해줄 error의 조건을 명시할 callback
-     */
-    ignoreError?: IgnoreErrorType;
-  }>
->;
-
-interface ErrorBoundaryState<ErrorType extends Error = Error> {
+type Props<ErrorType extends Error = Error> = {
+  /*
+   * @description 발생할 수 있는 error에 대한 기준값으로 이 값이 변경되면 error를 초기화합니다.
+   */
+  resetKeys?: unknown[];
+  onReset?(): void;
+  renderFallback: RenderFallbackType;
+  onError?(error: ErrorType, info: ErrorInfo): void;
+  /*
+   * @description 이 ErrorBoundary Context에서 처리하지 않고 throw해줄 error의 조건을 명시할 callback
+   */
+  ignoreError?: IgnoreErrorType;
+};
+
+interface State<ErrorType extends Error = Error> {
   error: ErrorType | null;
 }
 
-const initialState: ErrorBoundaryState = {
+const initialState: State = {
   error: null,
 };
 
-class BaseErrorBoundary extends Component<ErrorBoundaryProps, ErrorBoundaryState> {
+class BaseErrorBoundary extends Component<PropsWithRef<PropsWithChildren<Props>>, State> {
```

Comment:
How about maintain this lines as previous implemantation?

1. I think ErrorBoundaryProps is better name than Props 
2. ErrorBoundaryProps need to contain children

### [Discussion with react-error-boundary maintainer](https://github.com/bvaughn/react-error-boundary/pull/152)
It's not mandatory following react-error-boundary, but it's a bit sad to make this change only merging and not intuitive change.

Reply:
1. I think it's not important because the type isn't exported.
2. So it's wrapped by `PropsWithChildren`.